torch/_dynamo/variables/builder.py

```diff
@@ -140,6 +140,12 @@ def get_fake_examples(self):
             assert isinstance(
                 self.fake_tensor, torch._subclasses.fake_tensor.FakeTensor
             )
+            # For inplace ops changing the input's shape (unsqueeze_)
+            if (
+                not config.dynamic_shapes
```

Comment:
What happens with dynamic shapes?

Reply:
I added support for dynamic shapes in 953e13c